1984bot.py

```diff
@@ -52,7 +52,79 @@
 @bot.event
 async def on_ready():
     print('ONLINE')
-
+    
+async def on_message(self, message):
+    if message.author == client.user:
+       return
+     string = message.content
+     stringArr = ['place', 'triggers', 'here']
+     #All below replaces characters in a string (common substitutions) to prevent people from escaping the blacklist
+     string = re.sub('1', 'i', string)
+     string = re.sub('3', 'e', string)
+     string = re.sub('4', 'a', string)
+     string = re.sub('5', 's', string)
+     string = re.sub('ñ', 'n', string)
+     string = re.sub('7', 't', string)
+     string = re.sub('0', 'o', string)
+     string = re.sub('8', 'b', string)
+     string = re.sub('&', 'and', string)
+     string = re.sub('z', 's', string)

```

Comment:
this may not be such a good idea, if there are words on the blacklist that have a 'z' in them.

Reply:
Good point. Might be best to add the lines to convert S into Z right below it, which handles both cases in which a word has a Z replacing an S and an S replacing a Z.

Reply:
Problem with that tho is that something like `pizza`  would then match to `piss`, no? So idk

Reply:
Pizza would match with "Pissa", which is then converted back into "Pizza". A word with an S is converted to Z and back to S again, it works both ways I believe.